sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCast.scala

```diff
@@ -741,26 +785,23 @@ object GpuCast {
 
   private def castArrayToString(input: ColumnView,
       elementType: DataType,
-      ansiMode: Boolean,
-      legacyCastToString: Boolean,
-      stringToDateAnsiModeEnabled: Boolean): ColumnVector = {
+      options: CastOptions): ColumnVector = {
 
-    val (leftStr, rightStr) =  ("[", "]")
+    val (leftStr, rightStr) = ("[", "]")
```

Comment:
Should this be using `CastOptions.leftBracket`/`rightBracket` now, or is it intentional that this is unchanged? If so, it may be worth adding a comment explaining why

Reply:
No, array to string always use the square brackets

Reply:
will update with a comment